src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -200,6 +199,15 @@ export class SearchSource {
     return this.getField(field, false);
   }
 
+  /**
+   * Use ES response filtering to reduce JSON size
+   * @param filterPath Array of filter strings, such as ['*', '-**.key_as_string']
+   */
+  setFilterPath(filterPaths: string[]) {
+    this.filterPaths = filterPaths;
+    return this;
+  }
```

Comment:
Eventually it might be nice to expose a more generic way to directly add any param to a request, but a purpose-build method makes sense to me for the time being.

Reply:
Also could we add a unit test for this method, just to validate that the params make their way into the request?